### PR TITLE
Prevent DIV/0 revert calculating MOMP on a pool with no loans

### PR DESCRIPTION
**Pull request by EdNoepel**
# Description of change
## High level
Fix for #702 

# Description of bug or vulnerability and solution
Consumers should be able to calculate MOMP at any time, regardless of whether there is debt in the pool.  When there are no borrowers, we define MOMP as the HPB.  This means it will now be `MIN_PRICE` if there is no liquidity on the book.  There is no change to how MOMP is calculated for the first loan.

# Contract size
## Pre Change
============ Deployment Bytecode Sizes ============
  PoolInfoUtils            -  11,773B  (47.90%)

## Post Change
============ Deployment Bytecode Sizes ============
  PoolInfoUtils      -  11,828B  (48.13%)

# Gas usage
This call would only cost gas if used in a transaction from a proxy contract or outside of the Ajna protocol.
## Pre Change
| src/PoolInfoUtils.sol:PoolInfoUtils contract |                 |        |        |        |         |
|----------------------------------------------|-----------------|--------|--------|--------|---------|
| Function Name                                | min             | avg    | median | max    | # calls |
| momp                                         | 108324          | 108324 | 108324 | 108324 | 1       |

## Post Change
| src/PoolInfoUtils.sol:PoolInfoUtils contract |                 |        |        |        |         |
|----------------------------------------------|-----------------|--------|--------|--------|---------|
| Function Name                                | min             | avg    | median | max    | # calls |
| momp                                         | 34665           | 55698 | 36878  | 108353 | 9       |

### CODE REVIEW

**Review by grandizzy — APPROVED** [review]
LGTM

File: src/PoolInfoUtils.sol [hunk 1]
```diff
@@ -318,9 +319,10 @@ contract PoolInfoUtils {
     ) external view returns (uint256) {
         IPool pool = IPool(ajnaPool_);
 
-        (uint256 debt, , )       = pool.debtInfo();
         ( , , uint256 noOfLoans) = pool.loansInfo();
         noOfLoans += pool.totalAuctionsInPool();
+        if (noOfLoans == 0) return MAX_PRICE;
```
Comment (mattcushman) [inline src/PoolInfoUtils.sol]: What about returning the price of the top bucket?  Basically replace `Maths.wdiv(...))` below with `1`?

Comment (EdNoepel) [reply src/PoolInfoUtils.sol]: `MAX_PRICE` is the price of the top bucket.

Comment (mattcushman) [reply src/PoolInfoUtils.sol]: i mean top priced bucket with deposit > 0

Comment (EdNoepel) [reply src/PoolInfoUtils.sol]: Done in [e433d8cfba109191cdb67b74179836a7405d37d9](https://github.com/ajna-finance/contracts/pull/712/commits/e433d8cfba109191cdb67b74179836a7405d37d9); please re-review.

File: src/PoolInfoUtils.sol [hunk 2]
```diff
@@ -15,6 +15,7 @@ import {
     _priceAt,
     _reserveAuctionPrice,
     MAX_FENWICK_INDEX,
+    MAX_PRICE,
```
Comment (ith-harvey) [inline src/PoolInfoUtils.sol]: Why was `MAX_PRICE` added? I don't see it used, suggest it is removed.

**Review by ith-harvey — COMMENTED** [review]
Looks good, don't see issues with this addition, happy to approve once comments are addressed.

File: src/PoolInfoUtils.sol (hunk 2 above)
Comment (EdNoepel) [reply src/PoolInfoUtils.sol]: Removed in 111ce68448ee0054a82429f86e544b2b768f0908.

**Review by ith-harvey — APPROVED** [review]
LGTM, ship it!

**Pull request merged**